### DISCUSSION TITLE
Why is the donation app still available?

### DISCUSSION THREAD

**Discussion by beedell-roke** [opengisch/QField#4434] [Q&A]
![Screenshot_20230707-153726](https://github.com/opengisch/QField/assets/42837531/a311e4b5-bee6-4456-b1bf-e32af17d8528)

https://play.google.com/store/apps/details?id=ch.opengis.qfieldkarmaedition

**Comment by mbernasocchi** [comment]
Hi @rokejulianlockhart Thanks a lot for pointing out. I just unpublished the app. The best way to sponsor QField currently is via GitHub sponsoring at https://github.com/sponsors/opengisch

Cheers Marco
